### DISCUSSION TITLE
How to adjust the width of the right sidebar separately?

### DISCUSSION THREAD

**Discussion by vhtmui** [withastro/starlight#1830] [Q&A]
I try to modify the `--sl-sidebar-width` and `--sl-content-width`, but it affect the left sidebar and center section at the same time, and it only seems to affect the width of the text display, what I want to adjust is the width of the entire area.

**Comment by rvanhorn** [comment]
Hello,

I was testing this earlier myself and found targeting the following worked for me. 

```
.right-sidebar-panel > .sl-container
```

**Reply by vhtmui** [reply]
like this? I try it but nothing changes.
```
.right-sidebar-panel > .sl-container {
	width: 20%;
}
```
